draft-tigress-requirements.md

```diff
@@ -65,128 +70,123 @@ This document describes the use cases necessitating the secure transfer of digit
 
 # Introduction
 
-In this document we are identifying a problem of transferring digital credentials (e.g. a digital car key, a digital key to a hotel room or a digital key to a private house) from a wallet on one device (smartphone) to another, particularly, if these devices belong to two different platforms (e.g. one is iOS, another - Android).
-Today, there is no widely accepted way of transferring digital credentials securely between two digital wallets independent of hardware and software manufacturer. This document describes the problem space and the requirements for the solution the working group creates.
+In this document we are identifying a problem of transferring digital credentials (e.g. a digital car key, a digital key to a hotel room) from one device (e.g. smartphone) to another. Today, there is no widely accepted way of transferring digital credentials securely between two devices from different manufacturers. 
 
 A Working Group, called Tigress has been established to find a solution to the problem described above.
 Within the WG an initial solution was presented (https://datatracker.ietf.org/doc/draft-art-tigress). The community decided to generalize the requirements to the solution and consider alternative solutions within the WG.
 
-This document presents the general requirements to possible solutions and  specifies certain privacy requirements in order to maintain a high level of user privacy.
+This document presents the general requirements to possible solutions and specifies certain privacy requirements in order to maintain a high level of user privacy.
 
 
-# General Setting
+# Conventions and Definitions
 
-When sharing digital secure credentials, there are several actors involved. This document will focus on sharing information between two digital wallets, directly or through an intermediary server.
```

Comment:
I disagree that the entire lines 79 - 86 can be removed. This can be re-written in a more succinct manner, keeping the essence  that describes the actors and reasons to have them.
Eric in the IEFT review agreed this shall be preserved in the "General Layout / Settings" form

Reply:
Line 79 proposed a solution even before posing the problem.
Line 80 was blank
Line 81 described digital credential but did not provide a definition. It's not provided in new line 88
Line 82  is now described at Line 90 where the role of provisioning entity is clearly described.
Line 83 was blank
Line 84 is described at line 95.
line 85 was blank
line 86 is moved to line 100 where is made more sense in the context.

Effectively all those lines have been moved to places where they make most sense.

We are trying to create a draft that makes sense to the entire community. I'm sure Eric will see the value in the edits as well.